dashboard/src/modules/components/OverviewComponent/common-component.jsx

```diff
@@ -350,3 +365,24 @@ export const SelectDateComponent = (props) => (
     <DatePicker value={props.value} onChange={props.onDateSelect} />
   </div>
 );
+export const QuisbyResultsBtn = (props) => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const loadData = (newPageUrl) => {
+    const params = [{ name: props.name, rid: props.rId }];
+
+    dispatch(getQuisbyData(params, navigate));
+  };
+  return (
+    <Button
+      variant="link"
+      icon={<FileIcon />}
+      iconPosition="right"
+      onClick={() =>
+        loadData(`/dashboard/quisby-results/${props.name}/${props.id}`)
+      }
+    >
+      Quisby Results
```

Comment:
"Quisby Results" doesn't really mean anything. We're showing a summary benchmark graph. We labeled the API "visualize", but that's probably too generic for the UI label. Maybe "Graph result"?

Reply:
Updated this UI. It's no longer needed